yt_dlp/extractor/ilpost.py

```diff
@@ -23,10 +24,11 @@ class IlPostIE(InfoExtractor):
             'url': 'https://www.ilpost.it/wp-content/uploads/2023/12/28/1703781217-l-invasione-pt1-v6.mp3',
             'timestamp': 1703835014,
             'upload_date': '20231229',
+            'description': '<p>Circa tre miliardi di persone, oggi, parlano lingue che hanno un’unica antenata: dall’italiano all’inglese, passando per il farsi e l’islandese, queste lingue discendono tutte da una lingua arrivata in Europa circa cinquemila anni fa, insieme a un gruppo di persone ben preciso.<br />\nCon la loro lingua queste persone si portarono dietro anche alcuni oggetti, miti e leggende, e una certa visione della società, lasciando tracce indelebili ancora oggi.</p>\n<p>Per approfondire gli argomenti trattati nel podcast abbiamo raccolto in <a href="https://www.ilpost.it/2024/01/05/invasione-testi/?homepagePosition=3">questa pagina</a> le cose da leggere e da guardare dopo aver ascoltato le puntate.</p>\n',
```

Comment:
use `md5:` or `re:` instead of the whole long string. also looks like we need to `clean_html`